Tests/LiquidProjections.NEventStore.Specs/NEventStoreAdapterSpecs.cs

```diff
@@ -124,7 +125,7 @@ public When_requesting_a_subscription_beyond_the_highest_available_checkpoint()
                     var eventStore = A.Fake<IPersistStreams>();
                     A.CallTo(() => eventStore.GetFrom(A<string>.Ignored)).Returns(new[] {The<ICommit>()});

```

Comment:
Why would the event store return a commit that is earlier than the requested checkpoint?